README.md

```diff
@@ -463,19 +463,31 @@ require("mobile-cli-lib").devicesService.mapAbstractToTcpPort("4df18f307d8a8f1b"
 	});
 ```
 
-* `getApplicationsAvailableForDebugging(deviceIdentifier: string): Promise<string[]>` - This function checks the proc/net/unix file of the device for web views connected to abstract ports and returns the applications identifiers.
+* `getDebuggableApps(deviceIdentifier: string): Promise<Mobile.IAndroidApplicationInformation[]>` - This function checks the proc/net/unix file of the device for web views connected to abstract ports and returns information about the applications.
```

Comment:
On second glance, the interface's contents can be inferred from the sample result below, however I'd still prefer an explanation somewhere in this document, so that a reader may find the needed information by searching `IAndroidApplicationInformation` in the document, for example